bin/cli.js

```diff
@@ -1,15 +1,297 @@
 #!/usr/bin/env node
 
 const fs = require('fs-extra');
-const argv = require('minimist')(process.argv.slice(2));
-const childProcess = require('child_process');
-const inquirer = require('inquirer');
-const prompt = inquirer.createPromptModule();
-const util = require('util');
-const exec = util.promisify(childProcess.exec);
-const spawn = util.promisify(childProcess.spawn);
-const fork = util.promisify(childProcess.fork);
-
-let command = argv._[0];
-
-// TODO: Implement
+const ldposClient = require('ldpos-client');
+const { CliInterface } = require('@maartennnn/cli-builder');
+const actions = require('../lib/actions');
+const { FULL_CONFIG_PATH } = require('../lib/constants');
+
+const cli = new CliInterface({
```

Comment:
Can we rename this to just `CLI` or `REPLClient` or something else?
CLI stands for 'command line interface' so `CliInterface` is like `Command Line Interface Interface`.

Reply:
resolved in 8688b5e